core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
It's an addition to the pre-defined list of akka phases so it needs to be in the conf. But if someone changes the value it will mess up the order in which the phases are executed which we don't want.
The alternative would be to hardcode it like I did for the overriding part of the config. However like you mentioned in another comment we can apparently put it in the `application.conf`, which I'm ok with as long as it's absolutely clear that this will NOT work properly if someone does not include our `application.conf` in theirs (or copy the whole thing...).

I realize this might not make a lot of sense so we can TechTalk it if you want :)